.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
```

Comment:
You mean inside IDEA ?
Well, suppose your colleague doesn't have these settings by default (even you when installing new IDEA version might not reapply your old settings). When he checks out your project to work with it, he has to manually setup these settings in order to prevent commiting some unwanted stuff into the repo.